std_msgs/Header header
float32 azimuth
float32 elevation
float32 range
string asset_name